docs/releasenotes/release-1.126.md

```diff
@@ -22,10 +22,35 @@ TODO: list contributors with `git log v1.125.0... | grep Merge | grep from | awk
 
 ## Modified Beta Reconciliation
 
-We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
+We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The existing API is unchanged. To use the direct controller, add the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation to the corresponding CR object.
+
+* **`SecretManagerSecret`**
+
+  * #510 Enhanced `spec.rotation.nextRotationTime` to use a fixed datetime value to avoid relative `now()` friction. 
```

Comment:
I like this list of things we fixed, but is the bulleting correct?

Reply:
Here's the preview (from "View File"), do you have any suggestions on the format? 
<img width="1133" alt="Screenshot 2024-12-05 at 10 41 45 AM" src="https://github.com/user-attachments/assets/1032dabe-0aa0-4981-8a55-9d6e636d55b7">